doc/sourmash-sketch.md

```diff
@@ -0,0 +1,147 @@
+# `sourmash sketch` documentation
+
+Most of the commands in sourmash work with **signatures**, which contain information about genomic or proteomic sequences. Each signature contains one or more **sketches**, which are compressed versions of these sequences. Using sourmash, you can search, compare, and analyze these sequences in various ways.
+
+To create a signature with one or more sketches, you use the `sourmash sketch` command. There are three main commands:
+
+```
+sourmash sketch dna
+sourmash sketch protein
+sourmash sketch translate
+```
+
+The `sketch dna` command reads in **DNA sequences** and outputs **DNA sketches**.
+
+The `sketch protein` command reads in **protein sequences** and outputs **protein sketches**.
+
+The `sketch translate` command reads in **DNA sequences**, translates them in all six frames, and outputs **protein sketches**.
+
+## Quickstart
+
+### DNA sketches for genomes and reads
+
+To compute a DNA sketch for a genome, run:
+```
+sourmash sketch dna genome.fna
+```
+This will create an output file `genome.fna.sig` in the current directory, containing a single DNA signature for the entire genome, calculated using the default parameters.
+
+
+Sourmash can work with unassembled reads; run
+```
+sourmash sketch dna -p k=21,k=31,k=51,abund metagenome.fq.gz
+```
+to compute three abundance-weighted sketches at k=21, 31, and 51, for the given FASTQ file.
+
+### Protein sketches for genomes and proteomes
+
+Likewise,
+```
+sourmash sketch translate genome.fna
+```
+will output a protein sketch in `./genome.fna.sig`, calculated by translating the genome sequence in all six frames and then using the default protein sketch parameters.
+
+And
+```
+sourmash sketch protein -p k=25,scaled=500 -p k=27,scaled=250 genome.faa
+```
+outputs two protein sketches to `./genome.faa.sig`, one calculated with k=25 and scaled=500, the other calculated with k=27 and scaled=250.
+
+If you want to use different encodings, you can specify them in a few ways; here is a parameter string that specifies a dayhoff encoding for the k-mers:
+```
+sourmash sketch protein -p k=25,scaled=500,dayhoff genome.faa
+```
+
+## More detailed documentation
+
+### Input formats
+
+`sourmash sketch` auto-detects and reads FASTQ or FASTA files, either uncompressed or compressed with gzip or bzip2. The filename doesn't matter; `sourmash sketch` will figure out the format from the file contents.
+
+You can also stream any of these formats into `sourmash sketch` via stdin by using `-` as the input filename.
```

Comment:
Yes, that's true, but then the name of the sig is recorded as `-` which is really confusing when you compare a bunch of files. 

Also, should there be an example for how to do this?